packages/nimble-components/src/patterns/button/doc-strings.ts

```diff
@@ -0,0 +1,20 @@
+export const appearanceVariantDescription = `This attribute has no effect on buttons with a \`ghost\` appearance.
```

Comment:
this should also be moved to the tests folder since it just pertains to storybook tests

Reply:
Done